Lombiq.HelpfulLibraries.OrchardCore/TagHelpers/ShapeTagHelperBase.cs

```diff
@@ -48,7 +48,7 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
     {
         var shape = await _shapeFactory.CreateAsync(
             ShapeType ?? await GetShapeTypeAsync(context, output),
-            await GetViewModelAsync(context, output));
+            new { ViewModel = await GetViewModelAsync(context, output) });
```

Comment:
This was bugged but I didn't notice last time, because it was created for the `<google-tag>` but in the end we used Liquid instead of cshtml so it was only kept in for parity. Now it works correctly and the fix won't affect the Google Analytics tag, because it was using the part after the `??` [here](https://github.com/Lombiq/Helpful-Extensions/blob/661c48feb168a415196ce0079ba0bb022cdbc145/Lombiq.HelpfulExtensions/Views/GoogleTag.cshtml#L18-L22), while the Razor tag helper was supposed to use the part before.